### PR TITLE
update to 1.5.2

### PR DESCRIPTION
**Pull request by cbouss**
panel 1.5.2

**Destination channel:** main

### Links

- https://github.com/holoviz/panel/blob/v1.5.2/pyproject.toml

### Explanation of changes:

- udpdate to 1.5.2

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,55 +1,53 @@
...
-    - tqdm >=4.48.0
     - bleach
-    - typing_extensions
+    - typing-extensions
     - pandas >=1.2
     - packaging
+    - tqdm
   run_constrained:
     - holoviews >=1.18.0
+    - bokeh-fastapi ==0.1.0
```
Comment (philippjfr) [inline recipe/meta.yaml:50]: bokeh-fastapi was pretty broken, this would be better:

```suggestion
    - bokeh-fastapi ==0.1.1
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,55 +1,53 @@
...
-    - tqdm >=4.48.0
     - bleach
-    - typing_extensions
+    - typing-extensions
     - pandas >=1.2
     - packaging
+    - tqdm
   run_constrained:
     - holoviews >=1.18.0
+    - bokeh-fastapi ==0.1.1
```
Comment (ryanskeith) [inline recipe/meta.yaml:50]: According to the pyproject.toml, this is too restrictive https://github.com/holoviz/panel/blob/5ef8909aeacde7f0896f4b352e751d9a405a8867/pyproject.toml#L80.  Is this something we should change here or in the project?

**Review by ryanskeith — COMMENTED** [review]
LGTM, I do have a question about the bokeh-fastapi run constraint.

File: recipe/meta.yaml (hunk 2 above)
Comment (philippjfr) [reply recipe/meta.yaml:50]: I'd probably actually suggest both, let's pin to `bokeh-fastapi>=0.1.1,<0.2` here and I'll do the same in Panel.

Comment (cbouss) [reply recipe/meta.yaml]: added a commit to address this

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,55 +1,54 @@
...
     - requests
-    - tqdm >=4.48.0
     - bleach
-    - typing_extensions
+    - typing-extensions
     - pandas >=1.2
     - packaging
+    - tqdm
   run_constrained:
     - holoviews >=1.18.0
```
Comment (Jrice1317) [inline recipe/meta.yaml:49]:
```suggestion
    - holoviews >=1.16.0
```
https://github.com/holoviz/panel/blob/v1.5.2/pyproject.toml#L74

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,55 +1,54 @@
...
+    # set bokeh-fastapi bounds as discussed in https://github.com/AnacondaRecipes/panel-feedstock/pull/39
+    - bokeh-fastapi >=0.1.1,<0.2
```
Comment (Jrice1317) [inline recipe/meta.yaml:51]:
```suggestion
    - bokeh-fastapi >=0.1.0
```
https://github.com/holoviz/panel/blob/v1.5.2/pyproject.toml#L80

**Review by Jrice1317 — APPROVED** [review]
Outside of the minor questions on the differences in pinnings, LGTM!

File: recipe/meta.yaml (hunk 3 above)
Comment (cbouss) [reply recipe/meta.yaml:49]: To match what was released previously, we can keep it as it is.

File: recipe/meta.yaml (hunk 4 above)
Comment (cbouss) [reply recipe/meta.yaml:51]: Based on philip's comment, we can keep the upper bound.

**Pull request merged**